clang/unittests/StaticAnalyzer/ExprEngineVisitTest.cpp

```diff
@@ -27,6 +29,14 @@ void emitErrorReport(CheckerContext &C, const BugType &Bug,
   }
 }
 
+inline std::string getMemRegionName(const SVal &Val) {
```

Comment:
```suggestion
static std::string getMemRegionName(SVal Val) {
```

Have you considered using the Val.dump instead of rolling out something custom like this?

Reply:
You're right, that's probably simpler in this case.